src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -587,8 +586,13 @@ private void ParsePatchElement(XElement patchElement, out bool upToDate, out boo
                 var contentsAttribute = element.Attribute(ContentAttributeName);
                 if (contentsAttribute == null)
                 {
-                    _service._reportAndSwallowException(
```

Comment:
BTW, i'm all for doing this in a better way.  The point of this was that it allowed me to actually stringently test the error paths of SymbolSearch from tests.  i.e. tests could simulate different sorts of exceptions happening at different times, and could verify that hte engine would do the right thing.